recipe/meta.yaml

```diff
@@ -92,7 +92,5 @@ about:
 extra:
   recipe-maintainers:
     - rmax
-    - honnibal
-    - ines
-    - adrianeboyd
     - h-vetinari
+    - conda-forge/spacy
```

Comment:
With this (once rerendered) you'd remove your rights from all spacy feedstocks. This feedstock _defines_  what @conda-forge/spacy is, so you cannot use it here.

Reply:
Fixed in https://github.com/conda-forge/spacy-feedstock/commit/863036591f9223a74f7022cc35272c16b688ff5a

Reply:
Oh, that makes sense, sorry for the mistake!

Reply:
no worries :)